W10 | project-survey-vite | Xing Y. & Erika O.

xingyin2024 opened the pull request:
## Netlify link
https://survey-personality.netlify.app

## Collaborators
Xing Y. & Erika O.

xingyin2024 committed:
add - collaborator with Erika

xingyin2024 committed:
add - branch flow for test

xingyin2024 committed:
add - create components file and Header/QuestionOne/Result.jsx

xingyin2024 committed:
update - add content for test in all jsx

xingyin2024 committed:
update - style for body and text

xingyin2024 committed:
Merge branch 'xing' into dev

xingyin2024 committed:
update - content in header

xingyin2024 committed:
Merge branch 'dev' into xing

xingyin2024 committed:
update - header

Commit:
Update: header

xingyin2024 committed:
add & update - set up the foundation for multistepform

xingyin2024 committed:
add - button in Welcome

xingyin2024 committed:
add - comment in Welcome

xingyin2024 committed:
update Welcome

xingyin2024 committed:
pull from dev for setted up of foundation

Commit:
ClassNames for qa and styling

xingyin2024 committed:
update - Q1 and MultiStepForm

xingyin2024 committed:
update - content of Q2 & Q3 and fix some small bug in MultiStepForm

xingyin2024 committed:
[XY] merge and fix conflict in jsx between branches erika and xing

xingyin2024 committed:
[XY]update - put welcome into multistepform and update APP

xingyin2024 committed:
[XY] debug - modified the currentstep to show the welcome first and then the questions

xingyin2024 committed:
[XY]update - delete name attribute in Q1 input

xingyin2024 committed:
[XY]update - add className for question&input&button and adjust structure of Q1-3

xingyin2024 committed:
[XY]update - add className for previous button

xingyin2024 committed:
[XY]update - modify input text in Q3 and alert in MSF

xingyin2024 committed:
update

Commit:
merge conflicts

Commit:
update q3, q4, q5

Commit:
new file for q4, q4new

Commit:
update: styling

Commit:
update: styling

Commit:
update: styling buttons mobil, desktop

xingyin2024 committed:
[XY]update - button style, page height and Q4 file name

xingyin2024 committed:
update media query logic and fix small bug with text and button position and size

xingyin2024 committed:
add netlift link and collaborators in PR md

xingyin2024 committed:
update title in html

xingyin2024 committed:
update radio button style

xingyin2024 committed:
clean up the code

JennieDalgren reviewed (changes requested):
Great job with this project!

Clean up some un useful console logs and fix the indentation in you css files.
Also take a look at the accessibility score on Lighthouse. The required number is at least 95. You got 89. I think it mainly has to do with color contrasts of you grey buttons.

Good job with components and great that you've added comments regarding each components purpose and functionality. It looks like you really focused on understanding the React state and components this week.

Do the small fixes and keep up the good work!

xingyin2024 committed:
clean up, add validation status for buttons and alert message for each step

xingyin2024 commented:
I implemented a validation function for both the 'Next' step and form submission, displaying validation messages in red text below the disabled button instead of using alerts. The button design was updated to a more modern shape, and I added value="" to the first option in the dropdown to ensure the button re-disables if an empty selection is made.
